Docker.DotNet/Endpoints/ImageOperations.cs

```diff
@@ -51,7 +51,7 @@ internal ImageOperations(DockerClient client)
             }
 
             string path = string.Format(CultureInfo.InvariantCulture, "images/{0}/json", name);
-            DockerApiResponse response = await this.Client.MakeRequestAsync(new[] {NoSuchImageHandler}, HttpMethod.Get, path, null).ConfigureAwait(false);
+            DockerApiResponse response = await this.Client.MakeRequestAsync(new[] { NoSuchImageHandler }, HttpMethod.Get, path, null).ConfigureAwait(false);
```

Comment:
Odd. I would swear it was VSCode who removed them in the first place. I wonder if they recently made a change? Do what the auto formatter does.